src/trace_context.js

```diff
@@ -3,7 +3,7 @@
  */
 
 const asyncHooks = require('async_hooks');
-const { eventInterface } = require('epsagon');
+const { eventInterface, tracer: originalTracer } = require('epsagon');
```

Comment:
using "originalTracer" because "tracer" is used everywhere in this file and I wanted to minimize changes.